### ISSUE TITLE
Problem with test_labels.py, ValueError: Need more than 4 values to unpack

### ISSUE THREAD

**Issue by prckent** [QMCPACK/qmcpack#1127]
See e.g. https://cdash.qmcpack.org/CDash/buildSummary.php?buildid=30186 . This is the build for the "short" set of tests. There are many errors of this kind reported.

```
Adding this unit test
Adding test deterministic-unit_test_afqmc_matrix
Warning: test labeling failed.  Test labeling error output:
Traceback (most recent call last):
  File "/scratch/pk7/QMCPACK_CI_BUILDS_DO_NOT_REMOVE/qmcpack/tests/scripts/test_labels.py", line 467, in <module>
    full_test,qmc_cuda,enable_soa,qmc_complex,qmc_mixed = sys.argv[1:]
ValueError: need more than 4 values to unpack
```

**Comment by markdewing** [comment]
The command line listed in the CDash output doesn't seem to match the script in test_automation.
This command line looks like it is setting QMC_CUDA to an empty string, which would cause problems with the number of arguments to test_labels.py.

```
/home/pk7/apps/spack/opt/spack/linux-rhel7-x86_64/gcc-4.8.5/cmake-3.12.2-rjlilwnv7oceaw3rs2fthbu6c6ak4vah/bin/cmake" "-DCMAKE_C_COMPILER=mpicc" "-DCMAKE_CXX_COMPILER=mpicxx" "-DCMAKE_C_FLAGS=''" "-DCMAKE_CXX_FLAGS=''" "-DENABLE_GCOV:BOOL=false" "-DCMAKE_BUILD_TYPE:STRING=Release" "-DQMC_DATA='/data/pk7/QMC_DATA'" "-DQMC_CUDA=''" "-DQE_BIN=''" "-DBUILD_AFQMC='1'" "-DENABLE_TIMERS=1" "-GUnix Makefiles" "/scratch/pk7/QMCPACK_CI_BUILDS_DO_NOT_REMOVE/qmcpack" 
```

**Comment by prckent** [comment]
The script in test_automation is not current. I was planning to update it after onboarding the test labeling. 

This error looks to be a result of our ctest_script.cmake, which is current. It unconditionally adds unset variables such as QMC_CUDA to CTEST_OPTIONS.

**Comment by prckent** [comment]
Closed by #1141
